.gitignore

```diff
@@ -4,3 +4,7 @@ __pycache__
 *.csv
 .vscode
 *.swp
+Tests/Results
+
+!.vscode/LHR.code-workspace
+!.vscode/c_cpp_properties.json
```

Comment:
Just ignore the folder

Reply:
The exclamation marks are a whitelist. We specifically care about those files.